.github/ISSUE_TEMPLATE/Bug_report.md

```diff
@@ -0,0 +1,50 @@
+---
+name: Bug report
+about: Create a report to help us improve
+labels: bug
+
+---
+
+<!-- 🚨 STOP 🚨 STOP 🚨 STOP 🚨
```

Comment:
When triaging, one thing i realize that there are quite a number of users submitting bug reports of LB4 in `loopback` repo.  I wonder if it helps to put a comment here, something like: 
```
If you are opening bug reports for LoopBack 4, please go to: https://github.com/strongloop/loopback-next
```

Reply:
It makes me wonder if it would help to rename this GitHub repository to `loopback-legacy` as I have been proposing in the past.